stan/math/prim/mat/fun/matrix_exp_2x2.hpp

```diff
@@ -0,0 +1,42 @@
+#ifndef STAN_MATH_PRIM_MAT_FUN_MATRIX_EXP_2X2_HPP
+#define STAN_MATH_PRIM_MAT_FUN_MATRIX_EXP_2X2_HPP
+
+#include <unsupported/Eigen/MatrixFunctions>
+#include <stan/math/prim/scal/err/check_size_match.hpp>
+
+namespace stan {
+    namespace math {
+
+        using Eigen::Matrix;
+        using Eigen::Dynamic;
+
```

Comment:
You shouldn'tt put using statements at namespace level---you have to use the full qualified `Eigen::Matrix` and `Eigen::Dynamic` everywhere.  You can define using statements within function scope if you want.